terraform/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "get_an_identity_fqdn" {
+  value = "https://${azurerm_linux_web_app.auth-server-app.name}.azurewebsites.net"
```

Comment:
```suggestion
  value = "https://${azurerm_linux_web_app.auth-server-app.default_site_hostname}"
```

Reply:
`default_site_hostname` isn't exposed as an attribute. But `default_hostname` can be used. i will try and see what value it returns.